mobile/src/people/departments/departments-horizontal-scrollable-list.tsx

```diff
@@ -30,12 +32,80 @@ export class DepartmentsHScrollableList extends Component<DepartmentsHScrollable
         }
     }
 
+    public shouldComponentUpdate(nextProps: DepartmentsHScrollableListProps) {
+        if (this.currentPage > this.employeeCards.length) {
+            const { headDepartment } = this.props;
+            const subDepartments = headDepartment != null && headDepartment.children != null ? headDepartment.children : null;
+
+            if (this.props.employees.employeeIdsByDepartment.has(headDepartment.departmentId) && this.props.employees.employeeIdsByDepartment.get(headDepartment.departmentId).size > 0) {
+                const subordinates = this.props.employees.employeeIdsByDepartment.get(headDepartment.departmentId).map(x => this.props.employees.employeesById.get(x))
```

Comment:
Firstly `filter`, then `map`, and formatting please

Reply:
employeesIdsByDepartment contains just IDs so we aren't able to apply the filter first here.

Reply:
addressed